src/style.scss

```diff
@@ -1,14 +1,30 @@
-/* Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license */
-/* See LICENSE in the project root for license information. */
-
-/* --------------------------------------------------------------------- */
-
-/* stylelint-disable at-rule-no-unknown */
-
-/* Import Auro Sass variables */
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+/* stylelint-disable 
```

Comment:
Do we need to ignore all these rules and if so should we add them to the `.stylelintrc`?

Reply:
We do need to ignore all of these rules as of now until this repo gets a refactor. I was told that we should not edit the `.stylelintrc` as it is auto-generated from the generator.

Reply:
I see, that makes sense! I think in the future it would be nice to consolidate a lot of these rules to the `auro-library` and then we can use the local versions to override certain rules per-repo but we can discuss this another time!